lab.R

```diff
@@ -31,52 +31,72 @@ data_folder = file.path('data')
 #' # Problem 1 #
 #' 
 #' 1. *Load the file `7495_Robust-Intelligence.xlsx` using `readxl::read_excel()`.  Set the `n_max` argument to only read the first 25 rows.  Assign the input to `ri_df`.*
-#' 
+ri_df <- readxl::read_excel("C:\\Users\\panch\\OneDrive\\PhD Path\\UC Merced\\Year 3\\Data science\\Assignment\\Lab_4\\lab-w10-wrangling\\data\\7495_Robust-Intelligence.xlsx", n_max = 25 )
```

Comment:
You used a hard-coded path that doesn't exist on the Travis virtual machine.  Use `file.path()` instead.

Reply:
Done!